pkg/apis/submariner.io/v1/types.go

```diff
@@ -99,17 +99,16 @@ type LatencySpec struct {
 	MaxRTT uint64 `json:"maxRTT,omitempty"`
 	// +optional
 	StdDevRTT uint64 `json:"stddevRTT,omitempty"`
+}
 
-	// +optional
-	LastRTTInNanos uint64 `json:"lastRTTInNanos,omitempty"`
-	// +optional
-	MinRTTInNanos uint64 `json:"minRTTInNanos,omitempty"`
-	// +optional
-	AverageRTTInNanos uint64 `json:"averageRTTInNanos,omitempty"`
-	// +optional
-	MaxRTTInNanos uint64 `json:"maxRTTInNanos,omitempty"`
-	// +optional
-	StdDevRTTInNanos uint64 `json:"stddevRTTInNanos,omitempty"`
+// LatencySpec describes the round trip time information for a packet
+// between the gateway pods of two clusters.
+type LatencyRTTSpec struct {
+	Last    string `json:"last,omitempty"`
+	Min     string `json:"min,omitempty"`
+	Average string `json:"average,omitempty"`
+	Max     string `json:"max,omitempty"`
+	StdDev  string `json:"stddev,omitempty"`
```

Comment:
```suggestion
	StdDev  string `json:"stdDev,omitempty"`
```

Reply:
another option is to call the other field Stddev ... otherwise openapi will complain if we ever get it checked

Reply:
Done